buildingmotif-app/src/app/pointlabel-parser/pointlabel-parser.component.ts

```diff
@@ -69,9 +69,8 @@ export class PointlabelParserComponent implements OnInit {
     let fileReader = new FileReader();
     fileReader.onload = (e: any) => {
       // read file
-      const abbs: {s: string, type_name: string}[] = e.target.result.split("\n").map((row: string) => {
+      const abbs: {s: string, type_name: string}[] = e.target.result.split("\r\n").map((row: string) => {
```

Comment:
This won't necessarily work on all systems. https://stackoverflow.com/questions/21895233/how-to-split-string-with-newline-n-in-node